src/c/secrets-insecure.c

```diff
@@ -34,12 +34,12 @@ static iot_data_t *insecure_parse_config (iot_data_t *config)
     if (strncmp (key, SEC_PREFIX, SEC_PREFIXLEN) == 0)
     {
       size_t l = strlen (key);
-      if (strcmp (key + l - 5, "/path") == 0)
+      if (strcmp (key + l - 11, "/SecretName") == 0)
```

Comment:
I can guess at what the magic number 11 means but would prefer it was calculated somehow even if only in a comment. Also there seems to a define of SEC_PATH that was maybe related once?